tests/otel_tracing_e2e/test_e2e.py

```diff
@@ -2,13 +2,16 @@
 import os
 import time
 
-from utils import context, weblog, interfaces, scenarios, irrelevant, flaky
-from utils.tools import get_rid_from_request
+from utils import context, weblog, interfaces, scenarios, irrelevant
+from utils.tools import get_logger, get_rid_from_request
```

Comment:
a singleton is available: 

```suggestion
from utils.tools import logger, get_rid_from_request
```